community/community_support_process.md

```diff
@@ -0,0 +1,25 @@
+# Community Support Process
+
+## Monthly Community Tracker
+
+**Active tracker** -- [2021 - April Community Tracker](https://github.com/orgs/openebs/projects/31)
+
+The OpenEBS maintainers track community support issues using the monthly Community Tracker projects. The goal of the tracker is to lubricate the support process and gather feedback. It is expected that this project will lead to faster resolution and make OpenEBS user adoption easier.
+
+## Follow the steps below to keep the OpenEBS Community Tracker up to date with Slack:
```

Comment:
change this to: 

`The process for updating the OpenEBS community tracker are as follows:`

Reply:
Replace -- `issues` with `tickets`